js/src/block-editor/hooks/useMedia.js

```diff
@@ -67,6 +80,25 @@ const useMedia = ( fieldValue, onChange, allowedTypes ) => {
 		return getSettings()?.mediaUpload || legacyMediaUpload;
 	} );
 
+	useEffect( () => {
+		let timeout;
+		if ( ! media?.source_url ) {
+			const image = getImage( fieldValue );
+			if ( image ) {
+				setMedia( image );
+			} else {
+				timeout = setTimeout(
```

Comment:
This is kind of hacky. 

But sometimes the first request for the images fails without an error.

So this makes another request a second later if it failed.